docs/source/install.rst

```diff
@@ -65,6 +65,15 @@ Another way to achieve this is to pass the ``--recursive`` option to the ``git
 clone`` command which will automatically initialize and update DTKData in the
 DataTransferKit repository.
 
+You can also download the DTKData
+`repository <https://github.com/ORNL-CEES/DTKData>`_ yourself and then, link
+it into DTK directory:
+
+.. code::
+
+  $ cd $DTK_DIR
+  $ ln -s $DTKDATA_DIR DTKData
```

Comment:
Would need to define env variable `DTKDATA_DIR` for that to work.
I realize now that there is the same problem with `DTK_DIR` but the goal would be that people can almost copy/paste from these instructions.

Reply:
Sure but I assumed that people were smart enough to replace the `$DTKDATA_DIR` with the right path.

Reply:
I usually copy and paste without thinking and add `sudo` in front of it if it didn't work.

Reply:
I'll make sure to add `rm -rf /` at the end of the instruction then.

Reply:
```
root@00f6503f974c:/# rm -rf /
rm: it is dangerous to operate recursively on '/'
rm: use --no-preserve-root to override this failsafe
```